completions/_proxychains

```diff
@@ -0,0 +1,12 @@
+#compdef proxychains proxychains4
```

Comment:
why does the filename start with "_" ?

also i think this should just be called "proxychains4", not proxychains and then getting remapped for proxychains4.

Reply:
Hey! This is standard. Such as <https://github.com/soimort/you-get/tree/develop/contrib/completion> have three file: `_you-get` for zsh, `you-get-completion.bash` for bash, `you-get.fish` for fish. And the first line tell zsh this file not only complete proxychains, but also complete proxychains4.

Reply:
> And the first line tell zsh this file not only complete proxychains, but also complete proxychains4.

yes, and that's wrong. proxychains without 4 at the end is typically proxychains 3.1, and it has different command line options. if the distro packages proxychains-ng as "proxychains", which is not standard, then they also need to fix their shell completion accordingly.